Makefile

```diff
@@ -14,3 +14,6 @@ update-deps:
 
 install:
 	cd cmd/kruise && go install && cd -
+
+test:
+	go test pkg/**/*_test.go -v
```

Comment:
Take a look at the most recent Makefile changes, I think I would prefer that `test` over this just in case we add testing outside of the `pkg` scope.